### PR TITLE
Quantity issue in product shipment Issue #5315 Fixed

### PR DESCRIPTION
**Pull request by JunaidJaved248**
## Issue Reference
#5315 

## Description
As per demo video, While Admin try to save shipment, on that page Product qty showing there 1 instead of Exact Product Quantity. This issue occurred because while checking the product quantity from `product_inventories` in Product Model, Its returning bool value, If Product Quantity present in Product Inventories Table, its returning 1 that's why, its was showing 1 under Quantity Available in Shipment Page in Admin panel. 
So, I debug this issue, Changed the return type, Now Exact quantity is showing in create Shipment in admin Panel

## How To Test This?
1- Order Product From Customer Dashboard
2- After that view order from admin panel, proceed order processing save invoice, 
3- Click on ship button for creating shipment, then you will saw there, actual quantity will show under **Qty Available** instead of 1

### CODE REVIEW

**Comment by JunaidJaved248** [comment]
@devansh-webkul Test Got Failed, I think existing pull request that i created just before while fixing another issue. Files got merged. I debugged & Fixed The issue,
Let me know, I something wrong my end, I will create another pull request.
Thanks!!

**Comment by devansh-webkul** [comment]
Hi @JunaidJaved248,

I have seen you have written the `account-content` class code however these are fixing the issues but classes placement are not in the right place.

These all `account-content` classes, should be placed somewhere here, so that all `account-content` classes are organized,
https://github.com/bagisto/bagisto/blob/cda58a576f01bd48a77fd2028e8a461cd880e683/packages/Webkul/Velocity/src/Resources/assets/sass/components/app.scss#L762

File: packages/Webkul/Velocity/src/Resources/assets/sass/app.scss [hunk 1]
```diff
@@ -1136,4 +1136,10 @@ i.within-circle {
 
 .main-content-wrapper .content-list ul {
     width: 101.2% !important;
+}
+
+.account-content .account-layout .table table tr td {
+    width: auto;
+    border-top: none;
+    border-right: 1px solid #CCCCCC !important;
 }
```
Comment (devansh-webkul) [inline packages/Webkul/Velocity/src/Resources/assets/sass/app.scss:1145]: https://github.com/bagisto/bagisto/pull/5318#issuecomment-966026673

**Review by devansh-webkul — CHANGES_REQUESTED** [review]

**Comment by JunaidJaved248** [comment]
@devansh-webkul If I am not getting wrong, You mean that I have to make change in `bagisto/packages/Webkul/Velocity/src/Resources/assets/sass/components/app.scss` instead of `packages/Webkul/Velocity/src/Resources/assets/sass/app.scss`
Right!?

**Comment by devansh-webkul** [comment]
Just check the mentioned file, you will get an idea.

I want to say all classes should be organized there. As this will look much cleaner.

**Comment by JunaidJaved248** [comment]
@devansh-webkul Right Got it,
For now,  Should I need to organized current css classs that i wrote in other one that are already wrote in previous pull requests. Or Just need to organzied current changes in this pull request.

**Comment by devansh-webkul** [comment]
Only those classes which you have added and related to the `account` section.

**Comment by devansh-webkul** [comment]
- Still not fixed, just checked the mentioned path.
- And also resolve the conflict.

**Comment by devansh-webkul** [comment]
Everything is fine but why are you not writing here,
`bagisto/packages/Webkul/Velocity/src/Resources/assets/sass/components/app.scss `

The main class is already defined here, you just need to set your middle classes here.

**Comment by devansh-webkul** [comment]
See this till here its already added class,
https://github.com/bagisto/bagisto/blob/5bd7523d3b1d7d9ab6665c78fbef605617361466/packages/Webkul/Velocity/src/Resources/assets/sass/components/app.scss#L871

**Comment by JunaidJaved248** [comment]
> See this till here its already added class,
> 
> https://github.com/bagisto/bagisto/blob/5bd7523d3b1d7d9ab6665c78fbef605617361466/packages/Webkul/Velocity/src/Resources/assets/sass/components/app.scss#L871

I asked earlier about it, that should i have to write changes in that path or not?
Anyway, Let me make it done.
Now. I am going to make changes in that file okay.

**Comment by devansh-webkul** [comment]
Alright i am merging this thanks for the contribution. :rocket: :slightly_smiling_face:

**Pull request merged**